libbeat/outputs/elasticsearch/client.go

```diff
@@ -662,10 +670,13 @@ func (conn *Connection) Ping(timeout time.Duration) (string, error) {
 	return response.Version.Number, nil
 }
 
+// Close closes a connection.
+// TODO: it's a noop now
```

Comment:
Is this really a "TODO"? If so then let's say what needs to be done to make it complete.